lesson15/main.js

```diff
@@ -86,41 +88,49 @@ const displayModal= () => {
   overlay.style.display= "block";
 };
 
+const resetInput = () => {
+  const inputFields = document.getElementsByClassName("js-input");
+
+  for (errorMessage of errorMessages) {
+    errorMessage.textContent =""
+  };
+  for(inputField of inputFields) {
+    inputField.value = ""
+    inputField.style = "border border-slate-600 "
+  };
+}
+
 const closeModal = () => {
-  resetInput()
   modal.style.display="none";
   overlay.style.display="none";
+  resetInput();
 };
 
 const removeModal = () =>  document.getElementById("js-modal-wrapper").remove();
 
-const resetInput = () => {
-  numberInput.value = '';
-  validationMessage.style.display ="none";
-};
-
-const displayValidation = (message) => {
-  validationMessage.textContent = `${message}`;
-  validationMessage.style.display="block";
-}
-
-openListButton.addEventListener("click", () => {
-  const userInput = numberInput.value;
-  if(userInput === ''){
-    displayValidation("Input feild cannot be empty");
-    return;
-  }
-  if (!userInput.match(/^\d+$/)){
-    displayValidation("Input value must be a number");
-    return;
+const checkValidation = (id, serial, message) => {
+  if(id.value === "") {
+    errorMessages[serial].textContent = message;
```

Comment:
nits
The error message is specified as an array number, but in that case, I think the number would also need to be changed each time the input position is changed. 
Wouldn't it be easier to manage them later by giving them a unique id or class name?